Tools/autotest/win_sitl/jsbsimAPM_install.sh

```diff
@@ -11,3 +11,10 @@ cp src/JSBSim.exe /usr/local/bin
 cd ../ardupilot
 git submodule update --init --recursive
 ./modules/waf/waf-light configure --board=sitl
+
+read -p "Add $HOME/ardupilot/Tools/autotest to your PATH [Y/n]?"
+if [[ $REPLY =~ ^[Yy]$ ]]; then
+    echo 'export PATH=$PATH:$HOME/ardupilot/Tools/autotest' >> ~/.bashrc
```

Comment:
$ HOME does not become the installed path when installing to the D drive etc etc.
If you want to add a path, you'd better ask the user to specify the path.
Or, before executing git clone, I think that it is better to memorize the current current path and use that current path.

Reply:
It's seems that win_sitl does not support downloading outside the Cygwin directory.
It would be more convenient for the user not to specify the path because win_sitl automatically installs ardupilot in Cygwin directory's home.

Reply:
@tatsuy san comments thanks.
It is based on the assumption that it will be built in the default home directory that built Cygwin. For those who did not build it by default, or who do not want to set the path, enter n.

I think that this PR stumble once when checking operation on Windows. I also experienced. ardupilot If you are a PR including the experience of being a lecturer at Drone Software Engineer Training School, I think that it will reduce human errors.

Reply:
Thank you for your advice!